sandman/source/logger.h

```diff
@@ -42,5 +43,42 @@ bool LoggerAddMessage(char const* p_Format, ...);
 //
 // returns:		True if successful, false otherwise.
 //
-bool LoggerAddMessage(char const* p_Format, va_list& p_Arguments);
+[[gnu::format(printf, 1, 0)]] bool LoggerAddMessage(char const* p_Format,
+																	 std::va_list& p_Arguments);
 
+/**
+ * @brief Log an empty line. (The logger still prints the date and time.)
+ * 
+ * @note `LoggerAddMessage("")` triggers a warning `zero-length gnu_printf format string
+ * [-Werror=format-zero-length]`; this function accomplishes the same without triggering a warning.
+ * 
+ * @return `true` if successful, `false` otherwise
+ */
+inline bool LoggerAddEmptyLine()

```

Comment:
I'm not in love with pragmas to solve this. I think I would rather implement a real function for this.